net/nlbwmon/Makefile

```diff
@@ -1,13 +1,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=nlbwmon
-PKG_RELEASE:=3
+PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/jow-/nlbwmon.git
-PKG_SOURCE_DATE:=2020-04-11.1
-PKG_SOURCE_VERSION:=34a188769750927d839ad27fbfc90c973fe7bb06
-PKG_MIRROR_HASH:=75cd4e2eccb91256a91a3d48440cb347a696db64db2e71c050d284d20f98a77c
+PKG_SOURCE_DATE:=2021-09-01
+PKG_SOURCE_VERSION:=d82c910cc3d6021567c925e767ed1fd8c3934f4b
+PKG_MIRROR_HASH:=0ec86d002c514691d7ac94fcc5055f346fe5d255cc39598c36f5ef41fab86f0a
 
 CMAKE_INSTALL:=1
```

Comment:
This is pointless. include/cmake.mk already defaults to PKG_INSTALL:=1. I don't think there's anything that belongs in InstallDev here.